build/build.xml

```diff
@@ -835,6 +779,24 @@
     <exec executable="./linux/work/arduino" spawn="false" failonerror="true"/>
   </target>
 
+  <target name="build-arduino-builder" unless="no_arduino_builder">
+    <delete dir="${staging_folder}/arduino-builder-${platform}" includeemptydirs="true"/>
+    <mkdir dir="${staging_folder}/arduino-builder-${platform}"/>
+    <antcall target="untar">
+      <param name="archive_file" value="./arduino-builder-${platform}-${ARDUINO-BUILDER-VERSION}.tar.bz2" />
```

Comment:
For linuxarm the filename is `arduino-builder-linuxarm-1.3.23.tar.bz2`, but the file is actually called `arduino-builder-arm-1.3.23.tar.bz2`. I'm going to copy `arduino-builder-arm-1.3.23.tar.bz2` to `arduino-builder-linuxarm-1.3.23.tar.bz2` on the download server (and change the build script on jenkins to output the archive with `linuxarm` for the new releases).